src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/CollectionSection.js

```diff
@@ -214,7 +214,9 @@ class CollectionSection extends React.Component<Props> {
                         <div className={collectionSectionStyles.right}>
                             <ButtonGroup>
                                 {addable &&
-                                    <Button icon="su-plus" onClick={this.handleAddCollectionClick} />
+                                    <Button icon="su-plus" onClick={this.handleAddCollectionClick}>
+                                        Add folder
```

Comment:
We should use translation here. `t('sulu_media.add_collection')`. Would use this translation will check this again with the UX Team but think it should be consistent to the overlay title.